packages/types/src/index.ts

```diff
@@ -122,6 +122,21 @@ export type ChangelogFunctions = {
   getDependencyReleaseLine: GetDependencyReleaseLine;
 };
 
+export type GetAddMessage = (
+  changeset: Changeset,
+  commitOptions: null | Record<string, any>
+) => Promise<string>;
+
+export type GetVersionMessage = (
+  releasePlan: ReleasePlan,
+  commitOptions: null | Record<string, any>
+) => Promise<string>;
+
+export type CommitFunctions = {
+  getAddMessage: GetAddMessage;
+  getVersionMessage: GetVersionMessage;
```

Comment:
Since we allow those two to be optional~ I think this should be reflected in this type
```suggestion
  getAddMessage?: GetAddMessage;
  getVersionMessage?: GetVersionMessage;
```